pythonforandroid/archs.py

```diff
@@ -177,10 +184,11 @@ def get_env(self, with_flags_in_cc=True, clang=False):
 
         env['PATH'] = environ['PATH']
 
-        env['ARCH'] = self.arch
-        env['NDK_API'] = 'android-{}'.format(str(self.ctx.ndk_api))
-
+        # Specific flags for crystax (can we remove this?)
         if self.ctx.python_recipe and self.ctx.python_recipe.from_crystax:
+            # For crystax python, we can't use the host python headers:
+            env["CPPFLAGS"] += ' -I{}/sources/python/{}/include/python/'.\
+                format(self.ctx.ndk_dir, self.ctx.python_recipe.version[0:3])
             env['CRYSTAX_PYTHON_VERSION'] = self.ctx.python_recipe.version
```

Comment:
Yes I was also wondering, is CrystaX still supported/working?
It was dropped from the CI and I don't think any of the p4a maintainers would test things against CrystaX or even try to fix if it's not working.
So yes I would not bother adding theses lines, however I would not mix concerns too much would not address killing CrystaX here either.
Does this sound reasonable?

Reply:
> Yes I was also wondering, is CrystaX still supported/working?

I don't know if `crystax is still working, probably not...

>  I would not mix concerns too much would not address killing CrystaX here either.
Does this sound reasonable?

 Yes, I totally agree with you, I would keep this as simple as we can and I would do the `killing CrystaX` in another pr